cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -128,6 +131,12 @@ public class KafkaCluster extends AbstractModel implements SupportsMetrics, Supp
     protected static final String REPLICATION_PORT_NAME = "tcp-replication";
     protected static final int KAFKA_AGENT_PORT = 8443;
     protected static final String KAFKA_AGENT_PORT_NAME = "tcp-kafkaagent";
+
+    /**
+     * The configuration field name for Kafka metric reporters.
+     */
+    public static final String KAFKA_METRIC_REPORTERS_CONFIG_FIELD = "metric.reporters";
```

Comment:
It is not even used here. So why is it defined here?

Reply:
I defined it here because this is where the other Kafka config variables were defined. I will move it into the class where it is used and make it private.